keyboards/mechlovin/hannah910/rev1/readme.md

```diff
@@ -0,0 +1,15 @@
+# hannah910
+
+![hannah910](https://i.imgur.com/n8WN5Z7.jpg)
+
+Replacement PCB and plate for TGR 910 keyboard 
+
+Keyboard Maintainer: [Mechlovin'](https://github.com/mechlovin)  
```

Comment:
We use an unordered list for this section now.

`keyboards/mechlovin/hannah910/readme.md` should get these updates too.

```suggestion
* Keyboard Maintainer: [Mechlovin'](https://github.com/mechlovin)
```